File: workshop/angular-app/src/app/components/match-candidate/candidate.ts
export class Candidate {
  public id: number;
  public name: string;
  public avatar: string;
  public props: string[];

  constructor(id: number, name: string, avatar: string, props: string[]) {
    this.id = id;
    this.name = name;
    this.avatar = avatar;
    this.props = props;
  }
}

File: workshop/api-dotnet/Models/Candidate.cs
using System;
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace api_dotnet.Models
{
    public class Candidate
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public String id { get; set; }

        public String name { get; set; }
        public String avatar { get; set; }
        public String[] props { get; set; }
    }
}
